static/js/components/InviteNewGroupUserForm.jsx

```diff
@@ -70,10 +73,10 @@ const InviteNewGroupUserForm = ({ group_id }) => {
     });
   };
 
-  const toggleAdmin = (event) => {
+  const toggleCheckbox = (event) => {
     setFormState({
       ...formState,
-      admin: event.target.checked,
+      [event.target.name]: event.target.checked,
```

Comment:
Nice, clever way of generalizing this function. I probably would've defined two separate toggle functions haha